locales/puppetlabs-mysql.pot

```diff
@@ -140,35 +139,38 @@ msgid "`name` `parameter` must match user@host/table format."
 msgstr ""
 
 #: ./lib/puppet/type/mysql_grant.rb:57
-msgid "PROXY user not supported on mysql versions < 5.5.0. Current version %{version}"
+msgid ""
+"PROXY user not supported on mysql versions < 5.5.0. Current version "
+"%{version}."
 msgstr ""
 
 #: ./lib/puppet/type/mysql_grant.rb:67
-msgid "`table` `property` for PROXY should be specified as proxy_user@proxy_host ."
+msgid ""
+"`table` `property` for PROXY should be specified as proxy_user@proxy_host."
 msgstr ""
 
-#: ./lib/puppet/type/mysql_grant.rb:96 ./lib/puppet/type/mysql_user.rb:29
-msgid "Invalid database user %{user}"
+#: ./lib/puppet/type/mysql_grant.rb:96 ../lib/puppet/type/mysql_user.rb:29
```

Comment:
../lib/puppet/type/mysql_user.rb:29
needs to be 
./lib/puppet/type/mysql_user.rb:29
There are more below